src/config.cpp

```diff
@@ -382,7 +382,11 @@ namespace config {
     true  // dwmflush
   };
 
-  audio_t audio {};
+  audio_t audio {
+    {},  // audio_sink
+    {},  // virtual_sink
+    true,  // install_steam_drivers
```

Comment:
```suggestion
    false,  // install_steam_drivers
```

Default should probably be false, so we don't mess with anything until they opt in?